### ISSUE TITLE
Browser Testing and Tools Working Group Charter 2021

### ISSUE THREAD

**Issue by plehegar** [w3c/strategy#250]
New charter proposal, reviewers please take note.

# Charter Review

[Charter:](https://w3c.github.io/charter-drafts/browser-testing-tools.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [X] Existing WG recharter 
 - If this is a charter extension or revision, link a [diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2018%2F12%2Fbrowser-testing-tools.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2Fbrowser-testing-tools.html), and any issue discussion:

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security.  Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).

Communities suggested for outreach: 
  no outreach needed.

Known or potential areas of concern: 
  none (this is not deployed in user-facing products)

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)
  https://github.com/w3c/charter-drafts

Anything else we should think about as we review? 
  Nope

**Comment by michael-n-cooper** [comment]
No comments from APA; over to @brewerj to complete accessibility horizontal review.

**Comment by himorin** [comment]
Patent policy is pointing to 2017 but not 2020.

**Comment by plehegar** [comment]
> Patent policy is pointing to 2017 but not 2020.

Fixed in https://github.com/w3c/charter-drafts/commit/312cc8ea5746fbfd11e9db0aac9c24fa4f67167d

**Comment by plehegar** [comment]
cc @AutomatedTester in case you have comments

**Comment by AutomatedTester** [comment]
Looks good to me!

**Comment by sideshowbarker** [comment]
We just discussed the charter during this week’s W3C Strategy call, and one thing that was pointed out to me is, the charter doesn’t include some charter features that are now part of the current charter template. So I will either need to re-create it by essentially porting it over to the new template, or else (in the other direction) port over those missing charter features one-by-one into this.

@plh, you have a preference about which way I do it?

**Comment by himorin** [comment]
no comment/request from i18n.

**Comment by plehegar** [comment]
@sideshowbarker , no preference but I'd like to bring the charter to W3M next week, so please do the revamping before Tuesday.

**Comment by samuelweiler** [comment]
@plehegar Intriguingly, I suggested to @sideshowbarker that I would do the security and privacy reviews after he did the revamp, in part because at least two of the changes missed relate to security and privacy - and that's just from a first glance.

Since it sounds like your plan is incompatible with that ordering, I'll go ahead and flag this -needs-resolution, with the full list of items TBD.

**Comment by plehegar** [comment]
I did the revamping.

Same charter link as before https://w3c.github.io/charter-drafts/browser-testing-tools.html

@sideshowbarker , please make sure to wrap the horizontal review.

**Comment by sideshowbarker** [comment]
@samuelweiler This is ready for your review. And @brewerj I think we still need your review on this too

**Comment by samuelweiler** [comment]
Thank you @sideshowbarker.  It looks like https://github.com/w3c/charter-drafts/issues/329 and https://github.com/w3c/charter-drafts/issues/330 have been obviated by these changes.  https://github.com/w3c/charter-drafts/issues/331 still needs to be resolved.   I just filed two new issues: https://github.com/w3c/charter-drafts/issues/332 and https://github.com/w3c/charter-drafts/issues/333.

Moving to the new template resolves the substantive security and privacy concerns.  Marking these reviews as complete, though I encourage you to resolve the editorial issues above.

**Comment by sideshowbarker** [comment]
@samuelweiler Thanks much for the review and catches — I think the issues you spotted have now all been resolved.

@brewerj I believe this charter’s now only waiting for a11y review to be completed. To bring the charter to W3M this week, I need to get any necessary changes made to it before Tuesday morning US/East (late Tuesday evening my time).

**Comment by dontcallmedom** [comment]
[rechartered](https://lists.w3.org/Archives/Public/public-new-work/2021Mar/0011.html)
